src/Pages/Projects/Pagination.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
```

Comment:
also, Nice work with the pagination !
I got lost as to why we needed pagination on that section though, you probably make the good call

Nice work ! @Timothy-Achonu

Reply:
Thank you.